data/db-schema/src/main/resources/net/corda/db/schema/config/migration/config-creation-v1.0.xml

```diff
@@ -40,6 +42,21 @@
         <addPrimaryKey columnNames="section" constraintName="config_pk" tableName="config"
                        schemaName="${schema.name}"/>
 
+        <insert tableName="config" schemaName="${schema.name}">
+            <column name="section" value="corda.reconciliation"/>
+            <column name="config" value="
+                permissionSummaryIntervalMs=1000,
+                cpkWriteIntervalMs=1000,
+                cpiInfoIntervalMs=1000,
+                configIntervalMs=1000"/>
```

Comment:
These should be empty as the defaults will be applied by the default logic on first config reconciliation.

Reply:
Done